.github/ISSUE_TEMPLATE/acceptance-criteria-feature.yml

```diff
@@ -0,0 +1,79 @@
+name:  Feature/ Issue / Epic that requires acceptance criteria
+description: Create feature / issue /epic with clear requirements on the expected behavior of the Rancher UI
+title: "Summarize feature in one line"
+labels: ["kind/enhancement"]
+assignees: []
+body:
+- type: textarea
+  id: issue_scope
+  attributes:
+    label: Issue/Epic Scope
+    description: "A description of what the issue/epic is about and what it is being created for."
+    value: |
+      **Is your feature request related to a problem? Please describe.**
+      <!-- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] -->
+
+      **Describe the solution you'd like**
+      <!-- A clear and concise description of what you want to happen. -->
+
+      **Describe alternatives you've considered**
+      <!-- A clear and concise description of any alternative solutions or features you've considered. -->
+
+      **Additional context**
+      <!-- Add any other context or screenshots about the feature request here. -->
+  validations:
+    required: true
+- type: textarea
+  id: acceptance_criteria
+  attributes:
+    label: Acceptance Criteria
+    description: "List applicable use cases, each of them with a clear description of the expected behavior or alternative link to a requirements document. Please, try to include all possible use-cases. The QA team can help and include more as needed."
+    placeholder: |
+      List the steps or link sub-issues here using `- [ ] #issue_number` or URL. Progress will be tracked automatically.   
+      - [ ] 
+      - [ ]  
+  validations:
+    required: true    
+- type: textarea
+  id: uiux_dep
+  attributes:
+    label: UI/UX Dependencies
+    description: "List of sub-issues/steps required for the UI/UX team to complete the feature, accomplishing all the acceptance criteria."
+    placeholder: |  
+      List the steps or link sub-issues here using `- [ ] #issue_number` or URL. Progress will be tracked automatically.
+        - [ ] #issue_number or URL
+        - [ ] 
+  validations:
+    required: false        
+- type: textarea
+  id: other_dep
+  attributes:
+    label: Other Dependencies
+    description: "List of sub-issues/steps required for the backend or other teams to complete the feature, accomplishing all the acceptance criteria."
+    placeholder: |
+      List the steps or link sub-issues here using `- [ ] #issue_number` or URL. Progress will be tracked automatically.
+        - [ ]
+        - [ ] 
+  validations:
+    required: false
```

Comment:
I think we could combine these into one category, `Dependencies`. I'd also update the placeholder text before the checklist to be commented (similar to comment above).

Reply:
I believe that keeping the dependencies divided is a better approach so we can track work from our team and other teams separately.